code/modules/mob/living/carbon/carbon.dm

```diff
@@ -192,6 +192,8 @@
 			power_throw++
 		if(neckgrab_throw)
 			power_throw++
+		do_attack_animation(target, no_effect = 1) //PARIAH EDIT ADDITION - AESTHETICS
```

Comment:
`TRUE` rather than `1`, since it's a boolean.
```suggestion
		do_attack_animation(target, no_effect = TRUE) //PARIAH EDIT ADDITION - AESTHETICS
```